tests/langchain/sample_code/openai_agent.py

```diff
@@ -48,22 +47,25 @@ def multiply(a: int, b: int) -> int:
     return a * b
 
 
-llm = FakeOpenAI()
-tools = [add, multiply]
-prompt = ChatPromptTemplate.from_messages(
-    [
-        ("system", "You are a helpful assistant"),
-        MessagesPlaceholder("chat_history", optional=True),
-        ("human", "{input}"),
-        MessagesPlaceholder("agent_scratchpad"),
-    ]
-)
-agent = create_openai_tools_agent(llm, tools, prompt)
-agent_executor = AgentExecutor(
-    agent=agent,
-    tools=tools,
-    # Use env var to switch model configuration during testing
-    return_intermediate_steps=os.environ.get("RETURN_INTERMEDIATE_STEPS", False),
-)
+def create_openai_agent():
+    llm = FakeOpenAI()
+    tools = [add, multiply]
+    prompt = ChatPromptTemplate.from_messages(
+        [
+            ("system", "You are a helpful assistant"),
+            MessagesPlaceholder("chat_history", optional=True),
+            ("human", "{input}"),
+            MessagesPlaceholder("agent_scratchpad"),
+        ]
+    )
+    agent = create_openai_tools_agent(llm, tools, prompt)
+    return AgentExecutor(
+        agent=agent,
+        tools=tools,
+        # Use env var to switch model configuration during testing
+        return_intermediate_steps=os.environ.get("RETURN_INTERMEDIATE_STEPS", False),
```

Comment:
```suggestion
        return_intermediate_steps=os.environ.get("RETURN_INTERMEDIATE_STEPS", "false") == "true"
```

The default value for `os.environ.get` should be string.